scripts/sysdig-deploy/update-sysdig-deploy.sh

```diff
@@ -44,44 +51,50 @@ check_update_needed () {
             fi
         fi
     done
+
+    # update the subchart version in sysdig-deploy/Chart.yaml
+    if [[ $minor -gt 0 || $major -gt 0 || $patch -gt 0  ]]; then
+        bump_dep_preserve_blank "$chart" "$sysdig_subchart_version" "$new_subchart_version" "$sysdig_deploy_path"
+    fi
 }
 
 charts=( "$node_analyzer_chart_path" "$agent_chart_path" "$kspm_collector_chart_path" "$rapid_response_chart_path" "$admission_controller_chart_path" )
-for chart in ${charts[@]}
+for chart in "${charts[@]}"
 do
-    check_update_needed $chart
-done    
+    check_update_needed "$chart"
+done
 
-current_sysdig_deploy_version=$(yq eval '.version' charts/$sysdig_deploy_path/Chart.yaml)
+current_sysdig_deploy_version=$(yq '.version' charts/$sysdig_deploy_path/Chart.yaml)
 
 echo -e "\nsysdig-deploy version : $current_sysdig_deploy_version"
 IFS='.'
 read -ra sysdig_deploy_version <<< "$current_sysdig_deploy_version"
```

Comment:
can't this be rewritten as
```suggestion
IFS='.' read -ra sysdig_deploy_version <<< "$current_sysdig_deploy_version"
```
and avoid saving/restoring `IFS` altogether?

Reply:
yes, it's just that i started to fix a minor issue and i'm now refactoring it completely, anyway i agree